packages/atomic/src/components/facets/facet-header/facet-header.tsx

```diff
@@ -4,12 +4,14 @@ import ArrowBottomIcon from 'coveo-styleguide/resources/icons/svg/arrow-bottom-r
 import CloseIcon from 'coveo-styleguide/resources/icons/svg/close.svg';
 import {i18n} from 'i18next';
 import {Button} from '../../common/button';
+import {Heading} from '../../common/heading';
 
 export interface FacetHeaderProps {
   i18n: i18n;
   label: string;
   numberOfSelectedValues: number;
   isCollapsed: boolean;
+  headingLevel?: number;
```

Comment:
Should probably be non optional here, and have caller of this component pass in `0` if they don't need heading.